lm_eval/loggers/evaluation_tracker.py

```diff
@@ -112,12 +114,15 @@ def __init__(
         output_path: str = None,
         hub_results_org: str = "",
         hub_repo_name: str = "",
+        details_repo_name: str = "",
```

Comment:
Just noting that at some point it might be nice to mostly use `None` instead of `""` defaults here. Doesn't have to happen in this PR though!